modules/about-dynamic-plug-ins.adoc

```diff
@@ -30,5 +30,5 @@ When creating your plug-in, follow these general guidelines:
 
 * Prefix your CSS class names with your plug-in name to avoid collisions. For example, `my-plugin_\_heading` and `my-plugin_\_icon`.
 * Maintain a consistent look, feel, and behavior with other console pages.
-* Use link:https://github.com/openshift/enhancements/blob/master/enhancements/console/dynamic-plugins.md#localization[react-i18next] for localization.
+* Follow localization guidelines when creating your plug-in.
```

Comment:
Do we have those? I would at bare minimum I would link to the [dynamic-demo-plugin](https://github.com/openshift/console/tree/master/dynamic-demo-plugin#i18n).
Also the locales structure is demonstrated in there.